src/pvecontrol/sanitycheck.py

```diff
@@ -0,0 +1,196 @@
+import re
+
+from enum import Enum
+
+from pvecontrol.cluster import PVECluster
+from pvecontrol.storage import StorageShared
+from pvecontrol.utils import fonts
+
+class CheckType(Enum):
+  HA = 'HIGHT_AVAILABILITY'
+  Node = "NODE"
+
+class CheckCode(Enum):
+  CRIT = 'CRITICAL'
+  WARN = 'WARNING'
+  INFO = 'INFO'
+  OK = 'OK'
+
+ICONS = {
+  CheckCode.CRIT.value: '❌',
+  CheckCode.WARN.value: '⚠️',
+  CheckCode.INFO.value: 'ℹ️',
```

Comment:
`CheckCode.INFO` doesn't seem to be used, should we keep it anyway for a later use?

Reply:
I will remove it